lib/resources/base.js

```diff
@@ -66,7 +65,27 @@ module.exports = State.extend({
     }
   },
   /**
-   * prepare sending an event to all trackers that support events (ga, intercom).
+   * send an event to Stitch. This is the only actual call to the stitch tracker.
+   *
+   * @param {String} eventName   eventName to send
+   * @param {Object} metadata    attached metadata (5 keys max!)
+   * @api private
+   */
+  _send_stitch: function(eventName, metadata, callback) {
+    var tracker = this.trackers.get('stitch');
+    if (tracker && tracker.enabled) {
+      tracker.send(eventName, metadata);
+      if (callback) {
+        return callback(null, true);
```

Comment:
Optional: I would have found this easier to read it it had only one return statement:

```
let result = false;
...
if (tracker... ) {
  ...
  result = true;
}
if (callback) {
  return callback(null, result);
}
```

Reply:
I followed the pattern used by the other trackers, but you're right, this could be written cleaner. The entire metrics package could benefit from refactoring, but I didn't want to go as far and rewrite most of the code.

Reply:
Hmm, ok.